### PR TITLE
Update fluent-bit configuration to add logs in cloudwatch

### PR DESCRIPTION
**Pull request by lzianekhodja**

### CODE REVIEW

File: infrastructure/modules/monitoring/fluent-bit/configs/filter/filter-kubernetes-daemonset.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [FILTER]
     Name                kubernetes
-    Match               kube.*
+    Match               *
```
Comment (lemaitre-aneo) [inline infrastructure/modules/monitoring/fluent-bit/configs/filter/filter-kubernetes-daemonset.conf:3]: Could you explain why this match has been relaxed?

Comment (lzianekhodja) [reply infrastructure/modules/monitoring/fluent-bit/configs/filter/filter-kubernetes-daemonset.conf:3]: To use the same filte for seq and cloudwatch

File: infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf [hunk 2]
```diff
@@ -1,8 +1,21 @@
 [INPUT]
     Name               tail
     Tag                kube.*
-    Path               /var/log/containers/control-plane*.log, /var/log/containers/compute-plane*.log, /var/log/containers/armonik-metrics-exporter*.log, /var/log/containers/ingress*.log, /var/log/containers/keda-operator*.log, /var/log/containers/mongodb*.log
```
Comment (lemaitre-aneo) [inline infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf:4]: I fail to see why the conf has to split in 2, and more importantly, why the list of logs to watch is different.
Could you explain a bit?

Comment (lzianekhodja) [reply infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf:4]: The tag kube.* is for Seq and the tag application.* is for cloudwatch

Comment (lemaitre-aneo) [reply infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf:4]: I am not talking about the `Tag`, but about the `Path`

Comment (lemaitre-aneo) [reply infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf:4]: Excerpt of offline discussion:

> We don't want the same logs on cloudwatch than with seq.

Comment (lzianekhodja) [reply infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf:4]: We do not need logs of MongoDB, NGINX and Keda in Seq

**Review by lemaitre-aneo — APPROVED** [review]

**Pull request merged**